firebase-vscode/src/core/index.ts

```diff
@@ -53,9 +54,22 @@ export async function registerCore({
   });
 
   const emulatorsController = new EmulatorsController(broker);
+  // Start the emulators when the extension starts.
+  emulatorsController.startEmulators();
+
+  const openRcCmd = vscode.commands.registerCommand(
+    "firebase.openFirebaseRc",
+    () => {
+      for (const root of getRootFolders()) {
```

Comment:
I think you can use `getConfigPath()` from the same import file instead of looping over `getRootFolders()`

Reply:
I figured there's the case of multiroot workspaces.